packages/ngtools/webpack/package.json

```
@@ -28,13 +28,13 @@
   },
   "peerDependencies": {
     "@angular/compiler-cli": ">=10.0.0-next.0 < 11",
-    "typescript": ">=3.6 < 3.9",
+    "typescript": ">=3.9 < 4.0",
```

```suggestion
    "typescript": ">=3.9 < 3.10",
```
Just in case as well.

---

Same as above.